features/step_definitions/response_steps.rb

```diff
@@ -5,10 +5,16 @@
 end
 
 Given(/^I am logged in as (.+)$/) do |username|
-  visit '/users/sign_in'
-  fill_in 'user_email', with: username
-  fill_in 'user_password', with: 'password'
-  click_button 'Log in'
+  user = User.new
+  user.email = username
+  user.password = 'password'
+  user.save
+  Ability.new(user)
+
+  visit '/users//sign_in'
```

Comment:
(it's not but probably a minor typo) - nevermind this as it was removed in future commit in this PR